src/test/java/com/jcabi/github/RtLabelsITCase.java

```diff
@@ -70,9 +69,10 @@ public static void setUp() throws Exception {
         final Github github = new RtGithub(key);
         repos = github.repos();
         repo = repos.create(
-            Json.createObjectBuilder().add(
-                "name", RandomStringUtils.randomAlphanumeric(Tv.TEN)
-            ).add("auto_init", true).build()
+            new Repos.NewRepoSettings(
```

Comment:
@pinaf I plan to submit a follow-up PR for that and also for "give me a new MkRepo named `test`".